ui-participant/src/Navbar.tsx

```diff
@@ -199,3 +195,19 @@ export function CustomNavLink({ navLink }: { navLink: NavbarItem }) {
   }
   return <></>
 }
+
+/** the default join link -- will be rendered in the top right corner */
+export const getMainJoinLink = (portalStudies: PortalStudy[]) => {
+  const joinable = filterUnjoinableStudies(portalStudies)
+  /** if there's only one joinable study, link directly to it */
+  const joinPath = joinable.length === 1
+    ? `/studies/${joinable[0].study.shortcode}/join`
+    : '/join'
```

Comment:
I suppose it is possible that there are no joinable studies at a given time, but I assume that will be sorted out once the user gets to the join page?